compose/native/src/nativeMain/kotlin/androidx/compose/native/ComposeWindow.native.kt

```diff
@@ -55,17 +55,37 @@ class ComposeWindow(val parent: AppFrame) /*: SkiaWindow()*/ {
         NSWindowStyleMaskClosable or
         NSWindowStyleMaskResizable
 
-    private val nsWindow = NSWindow(
+    private val nsWindow = object : NSWindow(
         contentRect =  NSMakeRect(0.0, 0.0, 640.0, 480.0),
         styleMask = windowStyle,
         backing =  NSBackingStoreBuffered,
-        defer =  true)
+        defer =  true
+    ) {
+        override fun mouseDown(event: NSEvent) {
+            layer.owners.onMousePressed(event)
+            super.mouseDown(event)
+        }
+
+        override fun mouseUp(event: NSEvent) {
+            layer.owners.onMouseReleased(event)
+            super.mouseUp(event)
+        }
+
+        override fun mouseMoved(event: NSEvent) {
+            layer.owners.onMouseMoved(event)
+            super.mouseMoved(event)
+        }
+    }
 
 
     // private val clipMap = mutableMapOf<Component, ClipComponent>()
 
     init {
         nsWindow.contentView?.addSubview(layer.wrapped.nsView)
+        layer.wrapped.nsView.also {
+            // TODO: find the named constants (NSTrackingAreaOptions) and use them instead of 0x80 or 0x01 or 0x02
+            it.addTrackingArea(NSTrackingArea(it.frame, 0x80 or 0x01 or 0x02, it, null))
```

Comment:
`platform.AppKit.NSTrackingActiveAlways` `NSTrackingMouseEnteredAndExited` and freinds